### PR TITLE
Create images.tmp.env

### PR DESCRIPTION
**Pull request by lmilbaum**
**What this PR does / why we need it**:

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly

**Jira Ticket**:

```jira-ticket

```

**Release note**:

```release-note
Temp images.env file to be used by renovate

```

### CODE REVIEW

**Comment by lmilbaum** [comment]
No idea why sanity checks are failing

**Review by nunnatsa — COMMENTED** [review]
Shouldn't we have a comment with the version for each line?

**Comment by lmilbaum** [comment]
> Shouldn't we have a comment with the version for each line?

We should. TBH,  it is going to be a bit of challenge for me to identify the right version per line. It doesn't look that straight forward.

File: deploy/images.tmp.env [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+CDI_IMPORTER_IMAGE=quay.io/kubevirt/cdi-importer@sha256:72ee3dd7073a8c25c016f29d2075afe220a607e6081c50f48b0b0a16ddb516f6 # v1.58.0
+CDI_UPLOADPROXY_IMAGE=quay.io/kubevirt/cdi-uploadproxy@sha256:048f75f1223185b14824735ed2591fb4896ed53a6d12efec49e4ea2d79259e7d # v1.58.0
+CDI_UPLOADSERVER_IMAGE=quay.io/kubevirt/cdi-uploadserver@sha256:7bd32685cfacfac5879f1edec7da5cd563f78562fd58e6abb025bfc335c096b0 # v1.58.0
+HPPO_IMAGE=quay.io/kubevirt/hostpath-provisioner-operator@sha256:dd2948ca8e62d17eafba40bdcaad8367a97f092fd3b8d9849b2d62aec3152a72
+HPP_IMAGE=quay.io/kubevirt/hostpath-provisioner@sha256:adb1d33e95e8991dc47107926172469b5ecbeace509391ce021ca42d305a41d3
+HPP_CSI_IMAGE=quay.io/kubevirt/hostpath-csi-driver@sha256:8d84b9a33e0ec3749aece7c5f9879d1e0d248e8028ab3f776528a044d387033c
+MTQ_OPERATOR_IMAGE=quay.io/kubevirt/mtq-operator@sha256:ef5c4bb1fa72f9e6e75d5585ca352f7563720389e6c9876aa8b07d12d1be6194
+MTQ_CONTROLLER_IMAGE=quay.io/kubevirt/mtq-controller@sha256:96dd73ea0d7847d9f6322192559747bd85655e25ee25482d112e6e3c6fef21b2
+MTQ_LOCKSERVER_IMAGE=quay.io/kubevirt/mtq-lock-server@sha256:9614df0d5af7c998bcceaf73b548f2e3b29ff8ee4ca64c32613bfbdd6e452f6c
```
Comment (nunnatsa) [inline deploy/images.tmp.env:24]: `v1.1.11`

repo: https://github.com/kubevirt/managed-tenant-quota

File: deploy/images.tmp.env [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+NODE_DRIVER_REG_IMAGE=registry.k8s.io/sig-storage/csi-node-driver-registrar@sha256:0f64602ea791246712b51df334bbd701a0f31df9950a4cb9c28c059f367baa9e # v2.9.3
```
Comment (nunnatsa) [inline deploy/images.tmp.env:25]: `v2.9.3`

repo: https://github.com/kubernetes-csi/node-driver-registrar

File: deploy/images.tmp.env [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+SSP_OPERATOR_IMAGE=quay.io/kubevirt/ssp-operator@sha256:d57ef05fea09c4690dbe728ebe6a39d9df1a3574dfa7d18346bf3fe66afed702
+CDI_OPERATOR_IMAGE=quay.io/kubevirt/cdi-operator@sha256:8e92307f37470f5e83d234c6a894dfa97decbf362be96b504725ebf5494dccad # v1.58.0
+CDI_CONTROLLER_IMAGE=quay.io/kubevirt/cdi-controller@sha256:8cc9dc10cac91ad840fe51e0941593b9f15a2300d424fd6fa075bba22845e3ac # v1.58.0
+CDI_APISERVER_IMAGE=quay.io/kubevirt/cdi-apiserver@sha256:83dc33cdc7bf4b05e674c7f115dea5de74c057e420b62e37fe5a9228a5769006 # v1.58.0
+CDI_CLONER_IMAGE=quay.io/kubevirt/cdi-cloner@sha256:525ad1dcd27d7361b8f4cd57a9edf2b5ca68a0dfa3dd55b0946cabb04cc87827 # v1.58.0
+CDI_IMPORTER_IMAGE=quay.io/kubevirt/cdi-importer@sha256:72ee3dd7073a8c25c016f29d2075afe220a607e6081c50f48b0b0a16ddb516f6 # v1.58.0
+CDI_UPLOADPROXY_IMAGE=quay.io/kubevirt/cdi-uploadproxy@sha256:048f75f1223185b14824735ed2591fb4896ed53a6d12efec49e4ea2d79259e7d # v1.58.0
+CDI_UPLOADSERVER_IMAGE=quay.io/kubevirt/cdi-uploadserver@sha256:7bd32685cfacfac5879f1edec7da5cd563f78562fd58e6abb025bfc335c096b0 # v1.58.0
```
Comment (nunnatsa) [inline deploy/images.tmp.env:18]: repo: https://github.com/kubevirt/containerized-data-importer

File: deploy/images.tmp.env [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+HPPO_IMAGE=quay.io/kubevirt/hostpath-provisioner-operator@sha256:dd2948ca8e62d17eafba40bdcaad8367a97f092fd3b8d9849b2d62aec3152a72
+HPP_IMAGE=quay.io/kubevirt/hostpath-provisioner@sha256:adb1d33e95e8991dc47107926172469b5ecbeace509391ce021ca42d305a41d3
+HPP_CSI_IMAGE=quay.io/kubevirt/hostpath-csi-driver@sha256:8d84b9a33e0ec3749aece7c5f9879d1e0d248e8028ab3f776528a044d387033c
+MTQ_OPERATOR_IMAGE=quay.io/kubevirt/mtq-operator@sha256:ef5c4bb1fa72f9e6e75d5585ca352f7563720389e6c9876aa8b07d12d1be6194
+MTQ_CONTROLLER_IMAGE=quay.io/kubevirt/mtq-controller@sha256:96dd73ea0d7847d9f6322192559747bd85655e25ee25482d112e6e3c6fef21b2
+MTQ_LOCKSERVER_IMAGE=quay.io/kubevirt/mtq-lock-server@sha256:9614df0d5af7c998bcceaf73b548f2e3b29ff8ee4ca64c32613bfbdd6e452f6c
+NODE_DRIVER_REG_IMAGE=registry.k8s.io/sig-storage/csi-node-driver-registrar@sha256:0f64602ea791246712b51df334bbd701a0f31df9950a4cb9c28c059f367baa9e # v2.9.3
+LIVENESS_PROBE_IMAGE=registry.k8s.io/sig-storage/livenessprobe@sha256:82adbebdf5d5a1f40f246aef8ddbee7f89dea190652aefe83336008e69f9a89f
```
Comment (nunnatsa) [inline deploy/images.tmp.env:26]: `v2.11.0`

repo: https://github.com/kubernetes-csi/livenessprobe

File: deploy/images.tmp.env [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+CSI_SNAPSHOT_IMAGE=registry.k8s.io/sig-storage/csi-snapshotter@sha256:4c5a1b57e685b2631909b958487f65af7746361346fcd82a8635bea3ef14509d # v6.3.2
+CSI_SIG_STORAGE_PROVISIONER_IMAGE=registry.k8s.io/sig-storage/csi-provisioner@sha256:d078dc174323407e8cc6f0f9abd4efaac5db27838f1564d0253d5e3233e3f17f
```
Comment (nunnatsa) [inline deploy/images.tmp.env:28]: `v3.5.0`

repo: https://github.com/kubernetes-csi/external-provisioner

File: deploy/images.tmp.env [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+KUBE_RBAC_PROXY_IMAGE=quay.io/openshift/origin-kube-rbac-proxy@sha256:e2def4213ec0657e72eb790ae8a115511d5b8f164a62d3568d2f1bff189917e8
+KUBEVIRT_CONSOLE_PLUGIN_IMAGE=quay.io/kubevirt-ui/kubevirt-plugin@sha256:e79027973b09aac75860f267c7d7f830599978c38a081001f404b9a1c3d2990f
```
Comment (nunnatsa) [inline deploy/images.tmp.env:30]: `v4.14.0`

repo: https://github.com/kubevirt-ui/kubevirt-plugin

File: deploy/images.tmp.env [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+KUBEVIRT_CONSOLE_PROXY_IMAGE=quay.io/kubevirt-ui/kubevirt-apiserver-proxy@sha256:75f489a2cbc5bca0c2b7d9615e0cb5b93038c9423f939cff7b395268ae0bfcc9
```
Comment (nunnatsa) [inline deploy/images.tmp.env:31]: `v0.0.7`

repo: https://github.com/kubevirt-ui/kubevirt-apiserver-proxy

File: deploy/images.tmp.env [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+DIGEST_LIST="quay.io/kubevirt/virt-operator@sha256:0e6d56ea0ebdf9bb832f29ab0dc4603f0f619474bb8bc02a01e0a06e9d8d6f4a" # v1.1.0
```
Comment (nunnatsa) [inline deploy/images.tmp.env:32]: should be `v1.1.1`

File: deploy/images.tmp.env [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+HPPO_IMAGE=quay.io/kubevirt/hostpath-provisioner-operator@sha256:dd2948ca8e62d17eafba40bdcaad8367a97f092fd3b8d9849b2d62aec3152a72
+HPP_IMAGE=quay.io/kubevirt/hostpath-provisioner@sha256:adb1d33e95e8991dc47107926172469b5ecbeace509391ce021ca42d305a41d3
+HPP_CSI_IMAGE=quay.io/kubevirt/hostpath-csi-driver@sha256:8d84b9a33e0ec3749aece7c5f9879d1e0d248e8028ab3f776528a044d387033c
+MTQ_OPERATOR_IMAGE=quay.io/kubevirt/mtq-operator@sha256:ef5c4bb1fa72f9e6e75d5585ca352f7563720389e6c9876aa8b07d12d1be6194
+MTQ_CONTROLLER_IMAGE=quay.io/kubevirt/mtq-controller@sha256:96dd73ea0d7847d9f6322192559747bd85655e25ee25482d112e6e3c6fef21b2
+MTQ_LOCKSERVER_IMAGE=quay.io/kubevirt/mtq-lock-server@sha256:9614df0d5af7c998bcceaf73b548f2e3b29ff8ee4ca64c32613bfbdd6e452f6c
+NODE_DRIVER_REG_IMAGE=registry.k8s.io/sig-storage/csi-node-driver-registrar@sha256:0f64602ea791246712b51df334bbd701a0f31df9950a4cb9c28c059f367baa9e # v2.9.3
+LIVENESS_PROBE_IMAGE=registry.k8s.io/sig-storage/livenessprobe@sha256:82adbebdf5d5a1f40f246aef8ddbee7f89dea190652aefe83336008e69f9a89f
+CSI_SNAPSHOT_IMAGE=registry.k8s.io/sig-storage/csi-snapshotter@sha256:4c5a1b57e685b2631909b958487f65af7746361346fcd82a8635bea3ef14509d # v6.3.2
```
Comment (nunnatsa) [inline deploy/images.tmp.env:27]: repo:  https://github.com/kubernetes-csi/external-snapshotter

**Review by nunnatsa — COMMENTED** [review]
Added inline comments with repo addresses and versions

File: deploy/images.tmp.env [hunk 10]
```diff
@@ -0,0 +1,62 @@
+KUBEVIRT_OPERATOR_IMAGE=quay.io/kubevirt/virt-operator@sha256:0e6d56ea0ebdf9bb832f29ab0dc4603f0f619474bb8bc02a01e0a06e9d8d6f4a # v1.1.0
```
[the remark this answers is not in the document]
Comment (lmilbaum) [reply deploy/images.tmp.env:1]: Thanks. That is a great idea but I'd prefer it is added in a separate PR. I don't want to make mistakes.

**Pull request merged**